test_scripts/Defects/7_0/1903_Media_app_is_not_activated_during_active_embedded_navigation.lua

```diff
@@ -0,0 +1,59 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1903
+--
+-- Description:
+-- Media app is not activated during active embedded navigation.
+-- Precondition:
+-- SDL and HMI are started.
+-- In case:
+-- 1) Media app in LIMITED and AUDIBLE due to active embedded navigation.
+-- 2) SDL receives SDL.ActivateApp (<appID_of_media_app>) from HMI.
```

Comment:
```suggestion
-- 1) HMI activates embedded navigation. Media app is moved to LIMITED and AUDIBLE state.
-- 2) SDL receives SDL.ActivateApp (<appID_of_media_app>) from HMI.
```
Since they are really part of the same step, one results in the other